app/src/main/java/com/example/newsapp/ui/home/adapter/NewsAdapter.kt

```diff
@@ -40,10 +40,10 @@ class NewsAdapter :
         val article = differ.currentList[position]
         holder.itemView.apply {
             val binding: ArticleItemBinding = ArticleItemBinding.bind(holder.itemView)
-            if (!article.media.isNullOrEmpty()) {
+            article.media.firstOrNull()?.mediaMetadata?.firstOrNull()?.let{
                 Glide.with(holder.itemView).load(article.media[0].mediaMetadata[0].url)
```

Comment:
@jnandrawos use the "it" variable you got from let { } here

Reply:
done